cfgov/v1/models/base.py

```diff
@@ -58,15 +53,6 @@ def shared_q(self):
     def shared(self):
```

Comment:
Or, happy to pair on it ... maybe I just don't know how the PageManager is supposed to work.